scripts/validate_utils.py

```diff
@@ -0,0 +1,29 @@
+
+class ValidateError(Exception):
+    pass
+
+def validate(config):
+    # prototype based on <https://github.com/nextstrain/augur/blob/master/augur/validate.py>
+    from importlib import metadata
+    # <https://github.com/nextstrain/augur/issues/1358>
+    assert str(metadata.version("jsonschema")).startswith('3.'), "jsonschema must be version 3"
```

Comment:
Need to change this since [Augur 27.0.0](https://github.com/nextstrain/augur/releases/tag/27.0.0) [no longer supports v3](https://github.com/nextstrain/augur/pull/1691)